src/baseweb/BaseWebSelect.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import isPlainObject from 'lodash/isPlainObject';
+import isEqual from 'lodash/isEqual';
 import { Select, TYPE } from 'baseui/select';
 import { LayersManager } from 'baseui/layer';
```

Comment:
This will throw

Reply:
Replace  import { LayersManager } from 'baseui/layer';   with  import { LayersManager } from './LayersManager';